packages/volto-slate/src/editor/SlateEditor.jsx

```diff
@@ -261,6 +263,7 @@ class SlateEditor extends Component {
             editor={editor}
             initialValue={this.props.value || slateSettings.defaultValue()}
             onChange={this.handleChange}
+            label="teste"
```

Comment:
@Wagner3UB this can't be right

Reply:
@ichim-david How can a test left behind in a PR not be right? Are you joking?

I’ll fix it! =D